addons/crm/models/crm_team.py

```diff
@@ -723,6 +694,17 @@ def action_opportunity_forecast(self):
         action = self.env['ir.actions.actions']._for_xml_id('crm.crm_lead_action_forecast')
         return self._action_update_to_pipeline(action)
 
+    def action_open_unassigned_leads(self):
+        action = self.action_primary_channel_button()
+        context = action.get('context', '{}')
+        # preserve the original context to stick close to the main button action
+        # replace dynamic values so it can be evaluated as literals
+        context = context.replace('active_id', str(self.id)).replace('uid', str(self.env.user.id))
+        # inline the dict to avoid indentation issues
+        context = context.replace(' ', '').replace('\n', '')
+        action['context'] = literal_eval(context) | {'search_default_unassigned_leads': True}
```

Comment:
This is a bit low-level for me ... can't se use safe_eval with active_id / uid set in evaluation context so that we manipulate dicts and not strings ?